base/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Read the Docs - Sphinx Only Image
+#
+# This image only has a base sphinx and python environment.
+# It can't create PDF's and doesn't have special C libraries.
+
+FROM ubuntu:16.04
+MAINTAINER Read the Docs <support@readthedocs.com>
+LABEL version="latest"
+
+ENV DEBIAN_FRONTEND noninteractive
+ENV APPDIR /app
+ENV LANG C.UTF-8
+
+# System dependencies
+RUN apt-get -y update
+RUN apt-get -y install vim software-properties-common python-setuptools \
+    python3-setuptools python3 python3-pip python-dev python3-dev
```

Comment:
Why is this using the base python3 again? The latest image was using pyenv installed python.